source/5_1_release_notes.md

```diff
@@ -56,7 +56,7 @@ Rails 5.1 имеет встроенную поддержку для написа
 
 [Pull Request](https://github.com/rails/rails/pull/28038)
 
-Сейчас Rails позволяет управлять секретными данными приложения безопасным образом, созданным на основе гема [sekrets](https://github.com/ahoward/sekrets).
+Сейчас Rails позволяет управлять секретными данными приложения безопасным образом, вдохновленные гемом [sekrets](https://github.com/ahoward/sekrets).
```

Comment:
@Mik-die 
Rails now allows management of application secrets in a secure way, inspired by the sekrets gem.